packages/gatsby/src/utils/webpack-status.ts

```diff
@@ -0,0 +1,13 @@
+let isPendingStatus = false
```

Comment:
Why is it a separate module singleton and not part of, eg, Redux state?

Reply:
Good question. This is largely temporary and will be superseded by state machine coordination so didn't give it too much thought.

Reply:
That is fine then.